diff --git a/node_modules/native-base/src/basic/Tabs/DefaultTabBar.js b/node_modules/native-base/src/basic/Tabs/DefaultTabBar.js
index 9f3fad0..c70dfb5 100644
--- a/node_modules/native-base/src/basic/Tabs/DefaultTabBar.js
+++ b/node_modules/native-base/src/basic/Tabs/DefaultTabBar.js
@@ -25,6 +25,12 @@ const DefaultTabBar = createReactClass({
     tabStyle: PropTypes.shape({
       style: PropTypes.any,
     }),
+    tabStyle: PropTypes.oneOfType([
+      PropTypes.shape({
+        style: PropTypes.any,
+      },
+      PropTypes.array,
+    ]),
     renderTab: PropTypes.func,
     underlineStyle: PropTypes.shape({
       style: PropTypes.any,
diff --git a/node_modules/native-base/src/basic/Tabs/ScrollableTabBar.js b/node_modules/native-base/src/basic/Tabs/ScrollableTabBar.js
index ceb2597..b7dab37 100644
--- a/node_modules/native-base/src/basic/Tabs/ScrollableTabBar.js
+++ b/node_modules/native-base/src/basic/Tabs/ScrollableTabBar.js
@@ -30,12 +30,12 @@ const ScrollableTabBar = createReactClass({
     activeTextColor: PropTypes.string,
     inactiveTextColor: PropTypes.string,
     scrollOffset: PropTypes.number,
-    style: PropTypes.shape({
-      style: PropTypes.any,
-    }),
-    tabStyle: PropTypes.shape({
-      style: PropTypes.any,
-    }),
+    tabStyle: PropTypes.oneOfType([
+      PropTypes.shape({
+        style: PropTypes.any,
+      },
+      PropTypes.array,
+    ]),
     tabsContainerStyle: PropTypes.shape({
       style: PropTypes.any,
     }),
